frontend/zc_messaging/src/pages/DmBrowser/index.jsx

```diff
@@ -0,0 +1,10 @@
+import React from "react"
```

Comment:
is dm browser not the same as "all-dms" which has already been initialized

Reply:
I had it before you added all-dms folder.

I didn't remove it because I preferred the name *dms-browser* to *all-dms*

Reply:
okay... that means we will move the content of "all-dms" into "dm-browser"